Makefile

```diff
@@ -53,7 +54,7 @@ cluster/prepare:
 	-kubectl create -n $(NAMESPACE) -f deploy/role.yaml
 	-kubectl create -n $(NAMESPACE) -f deploy/role_binding.yaml
 	-kubectl apply  -n $(NAMESPACE) -f deploy/crds/mdc_v1alpha1_mobiledeveloperconsole_crd.yaml
-	-kubectl apply  -n $(NAMESPACE) -f deploy/crds/mdc_v1alpha1_mobileclient_crd.yaml
+	-kubectl apply  -n $(NAMESPACE) -f deploy/mdc_v1alpha1_mobileclient_crd.yaml
```

Comment:
Myself and @laurafitzgerald were talking about this just now, and we think that it would be eventually good to have this be generated the same as the others (i.e. `operator-sdk new api`, but not `operator-sdk new controller`). That doesn't need to be done as part of this task though, we can have another one for it :+1:

Reply:
if it does not get too much your time. I would use the operator-sdk tool as follows just to ensure that all that was required was created properly and cut and past the impl for there.

Reply:
> eventually good to have this be generated

Yeah, this is a good idea.
I prefer doing it separately.

> The API added was mdc.aerogear.org/v1alpha1. Would not be bettermobile-developer-console-operator.aerogear.com/v1alpha1?

I don't have a strong opinion. It sounds a bit too long. I would also do this separately later.

Reply:
+1 for the existing shorter one.